wagtail_localize/tests/test_translation_model.py

```diff
@@ -218,7 +218,7 @@ def setUp(self):
         self.fr_locale = Locale.objects.create(language_code="fr")
 
         self.page = create_test_page(title="Test page", slug="test-page", test_charfield="This is some test content")
-        self.source = TranslationSource.from_instance(self.page)
+        self.source, created = TranslationSource.get_or_create_from_instance(self.page)
```

Comment:
Confirming that `created` doesn't need to be `self.created`, correct?

Reply:
That's correct. We don't actually use this value here